dist/s_export

```diff
@@ -29,6 +29,8 @@ check()
     # Functions beginning with __ut are intentionally exposed to support unit testing when
     # Wiredtiger is compiled with HAVE_UNITTEST=1.
     grep -E -v '^__ut' |
+    # __wrap functions suppress MSan false positives. They're only present when compiled with MSan.
+    grep -E -v '^__wrap_' |
```

Comment:
Is there benefit in making this more specific, ie: `__wrap_f?stat`. Not 100% certain on the optional `f` syntax.

Reply:
Updated in [bf2fff0](https://github.com/wiredtiger/wiredtiger/pull/10685/commits/bf2fff030898bd9da6db65ddeb5f6f77afd03730)[bf2fff0](https://github.com/wiredtiger/wiredtiger/pull/10685/commits/bf2fff030898bd9da6db65ddeb5f6f77afd03730)[bf2fff0](https://github.com/wiredtiger/wiredtiger/pull/10685/commits/bf2fff030898bd9da6db65ddeb5f6f77afd03730)